packages/patternfly-4/react-charts/src/components/ChartDonut/examples/ChartDonut.md

```diff
@@ -62,15 +64,15 @@ import { ChartDonut } from '@patternfly/react-charts';
       width={350}
     />
   </div>
-</div>
+)
 ```
 
-```js title=Multi-color-(ordered)-donut-chart-with-right-aligned-legend
+```js title=Multi--color-right--aligned-legend
```

Comment:
Again, "ordered" is important. The title was "Multi-color (ordered) donut chart with right aligned legend"